### ISSUE TITLE
[CI] body.license.uid is null in reference/rest-api/info/line_45

### ISSUE THREAD

**Issue by droberts195** [elastic/elasticsearch#56106]
https://gradle-enterprise.elastic.co/s/f6y6ws73omhxi failed because there was no `license` section in the X-Pack info response:

```
[2020-05-03T15:08:23,062][INFO ][o.e.s.DocsClientYamlTestSuiteIT] [test] Stash dump on test failure [{
  "stash" : {
    "body" : {
      "build" : {
        "hash" : "293ff2f60eb1fce9af59d23692d783ad040e0654",
        "date" : "2020-05-03T18:56:21.156730Z"
      },
      "license" : null,
      "features" : {
        "analytics" : {
          "available" : true,
          "enabled" : true
        },
        "ccr" : {
          "available" : true,
          "enabled" : true
        },
        "enrich" : {
          "available" : true,
          "enabled" : true
        },
        "eql" : {
          "available" : true,
          "enabled" : true
        },
        "frozen_indices" : {
          "available" : true,
          "enabled" : true
        },
        "graph" : {
          "available" : true,
          "enabled" : true
        },
        "ilm" : {
          "available" : true,
          "enabled" : true
        },
        "logstash" : {
          "available" : true,
          "enabled" : true
        },
        "ml" : {
          "available" : true,
          "enabled" : true
        },
        "monitoring" : {
          "available" : true,
          "enabled" : true
        },
        "rollup" : {
          "available" : true,
          "enabled" : true
        },
        "security" : {
          "available" : true,
          "enabled" : false
        },
        "slm" : {
          "available" : true,
          "enabled" : true
        },
        "spatial" : {
          "available" : true,
          "enabled" : true
        },
        "sql" : {
          "available" : true,
          "enabled" : true
        },
        "transform" : {
          "available" : true,
          "enabled" : true
        },
        "vectors" : {
          "available" : true,
          "enabled" : true
        },
        "voting_only" : {
          "available" : true,
          "enabled" : true
        },
        "watcher" : {
          "available" : true,
          "enabled" : true
        }
      },
      "tagline" : "You know, for X"
    }
  }
}]
```

The failure message was:

```
java.lang.RuntimeException: Failure at [reference/rest-api/info:16]: stashed value not found for key [body.license.uid]
```

The repro command is:

```
./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/rest-api/info/line_45}" -Dtests.seed=F16A790D7469E4B3 -Dtests.security.manager=true -Dtests.locale=no-NO -Dtests.timezone=America/Detroit -Dcompiler.java=14 -Druntime.java=11
```

This did not reproduce locally for me.

**Comment by tvernum** [comment]
@ywangd Do you think this is fixed by #60498 or do we need to add something else as well?

**Comment by ywangd** [comment]
Unfortunately, this is another variant of the same underlying problem. It needs its own fix similar to `get-license` doc test. I'll raise a PR. Thanks for bringing this up.
